### ISSUE TITLE
[TW-540] Inputting accented characters interferes with zsh completion

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#575]
_Paul Beckingham on 2012-04-01T15:37:59Z says:_ 

 15:04:21 < efaistos> yeah i just twittered about a thing about accent when you use something like
15:04:35 < efaistos> task add "some french accent like é or à " project:
15:04:50 < efaistos> and then the completion doesnt work
15:05:02 < efaistos> for project or tags after that ...
15:05:09 < pbecking1am> efaistos: that's interesting.  What is your shell?
15:05:25 < efaistos> zsh
15:05:40 < pbecking1am> Thanks.  Which taskwarrior do you have?
15:05:46 < efaistos> 2.0 from source

Thanks to Hyde Stevenson

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-04-01T15:37:59Z
Modified: 2014-02-09T02:04:41Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-06-10T20:30:21Z says:_ 

 Inadvertently fixed during #1006 work.
